common/djangoapps/student/management/commands/bulk_change_enrollment_csv.py

```diff
@@ -0,0 +1,98 @@
+"""
+Management command to change many user enrollments in many courses using
+csv file.
+"""
+import csv
+import logging
+from os import path
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from student.models import CourseEnrollment, User
+from student.models import CourseEnrollmentAttribute
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Command(BaseCommand):
+    """
+        Management command to change many user enrollments in many
+        courses using the csv file
+        """
+
+    help = """
+        Change the enrollment status of all the users specified in
+        the csv file in the specified course to specified course
+        mode.
+        Could be used to update Effected users by order
+        placement issues. If number of students is a lot for the
+        different courses.
+        Similar to bulk_change_enrollment but uses the csv file
+        input format and can enroll students in multiple courses.
+
+        Example:
+            $ ... bulk_change_enrollment_csv csv_file_path
+        """
+
+    def add_arguments(self, parser):
+        """ Add argument to the command parser. """
+        parser.add_argument(
+            '--csv_file_path',
+            required=True,
+            help='Csv file path'
+        )
+
+    def handle(self, *args, **options):
+        """ Main handler for the command."""
+        file_path = options['csv_file_path']
+
+        if not path.isfile(file_path):
+            raise CommandError("File not found.")
+
+        with open(file_path, 'rb') as file:
+            file_reader = csv.DictReader(file)
+            for row in file_reader:
+                try:
+                    course_key = CourseKey.from_string(row['course_id'])
+                except InvalidKeyError:
+                    raise CommandError('Invalid or non-existant course id {}'.format(row['course_id']))
+
+                try:
+                    user = User.objects.get(username=row['user'])
+                except:
+                    raise CommandError('Invalid or non-existant user {}'.format(row['user']))
+
+                try:
+                    # Student might be or not enrolled in course
+                    course_enrollment = CourseEnrollment.get_or_create_enrollment(user=user, course_key=course_key)
+
+                    if course_enrollment.mode == row['mode']:
+                        logger.info("Student [%s] is already enrolled in Course [%s] in mode [%s].", user.username,
```

Comment:
I think this message will make more sense if we use just get enrollment instead of get_or_create. This condition will be evaluated every time even when new enrollment will be created by this command it self.